src/core_modules/capture-core/components/List/OnlineList/OnlineList.component.js

```diff
@@ -62,6 +62,9 @@ const getStyles = (theme: Theme) => ({
     loadingCell: {
         textAlign: 'center',
     },
+    loader: {
+        display: 'inline-block',
```

Comment:
A bit out of scope, but this is just ensuring that the spinner is aligned in the center (horizontally)